packages/ifpack2/src/Ifpack2_Details_Factory_def.hpp

```diff
@@ -189,6 +184,31 @@ create (const std::string& precType,
   return prec;
 }
 
+
+template<class SC, class LO, class GO, class NT>
+bool
+Factory<SC, LO, GO, NT>::
+isSupported (const std::string& precType)
+{
+  // precTypeUpper is the upper-case version of precType.
+  std::string precTypeUpper = canonicalize(precType);
+
+  std::vector<std::string> supportedNames = {
+    "SCHWARZ",
```

Comment:
Huh, [apparently it's OK](https://godbolt.org/z/Ld3iz6) to have the comma there even if `"SUPPORTGRAPH"` is `#ifdef`'d out.